src/tendril.rs

```diff
@@ -963,22 +965,16 @@ impl<F, A> Tendril<F, A>
     // By and large they shouldn't need to worry about the distinction at all,
     // and going out of your way to make it owned is pointless.
     #[inline]
-    fn make_owned(&mut self) {
-        let len = self.len32();
-        if len > MAX_INLINE_LEN as u32 {
-            unsafe {
-                self.make_owned_with_capacity(len);
-            }
+    unsafe fn make_owned(&mut self) {
```

Comment:
I marked this `unsafe` because `make_owned_with_capacity` was, but I don’t know if either really can introduce memory unsafety.